nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-2-6-processors/src/main/java/org/apache/nifi/processors/kafka/pubsub/ConsumeKafkaRecord_2_6.java

```diff
@@ -541,8 +541,10 @@ public void onTrigger(ProcessContext context, ProcessSession session) throws Pro
                     + "Will roll back session and discard any partially received data.", lease);
             } catch (final KafkaException kex) {
                 getLogger().error("Exception while interacting with Kafka so will close the lease {} due to {}", lease, kex, kex);
```

Comment:
This is a good opportunity to remove the duplicative `due to` portion of the log, since it is already included in the stack trace.
```suggestion
                getLogger().error("Exception while interacting with Kafka so will close the lease {}", lease, kex);
```